pkg/model/awsmodel/iam.go

```diff
@@ -333,10 +333,7 @@ func (b *IAMModelBuilder) buildIAMTasks(role iam.Subject, iamName string, c *fi.
 			}
 
 			c.AddTask(t)
-		}
 
-		// Generate additional policies if needed, and attach to existing role
-		if !shared {
```

Comment:
Does it hurt in any way to leave this as 2 separate blocks of code?

Reply:
`iamRole` is defined in one and used in the other.

Reply:
Maybe just create 2 blocks and keep the original comments for each block?

Reply:
yeah removing the comments were unecessary.